epam-promo/components/filters/FiltersToolbarItem.tsx

```diff
@@ -75,7 +75,7 @@ const FiltersToolbarItemImpl = (props: FiltersToolbarItemProps) => {
         <DropdownContainer>
             <Panel background="white">
                 { renderHeader() }
-                { <FilterItemBody { ...props } { ...dropdownProps } value={ getValue() } onValueChange={ onValueChange }/> }
+                { <FilterItemBody { ...props } { ...dropdownProps } currentPredicate={ { predicate, name: predicateName } } value={ getValue() } onValueChange={ onValueChange }/> }
```

Comment:
better rename currentPredicate to selectedPredicate

Reply:
done